zerver/lib/events.py

```diff
@@ -441,6 +441,8 @@ def name(sub):
     elif event['type'] == "update_display_settings":
         if event['setting_name'] == "twenty_four_hour_time":
             state['twenty_four_hour_time'] = event["setting"]
+        if event['setting_name'] == "high_contrast_mode":
+            state['high_contrast_mode'] = event["setting"]
```

Comment:
Oh, never mind, this is a further step we should be cleaning up; I was looking in the wrong part of the file.